crates/bevy_render/src/render_phase/draw.rs

```diff
@@ -159,6 +159,12 @@ impl<P: PhaseItem> DrawFunctions<P> {
 pub trait RenderCommand<P: PhaseItem> {
     /// Specifies the general ECS data (e.g. resources) required by [`RenderCommand::render`].
     ///
+    /// Due to lifetime limitations of the `Deref` trait,
+    /// [`SRes::into_inner`](bevy_ecs::system::lifetimeless::SRes::into_inner) must be called on
```

Comment:
```suggestion
    /// [`SRes::into_inner`] must be called on
```

Reply:
I will be using this style from now on for long paths, thanks.